### DISCUSSION TITLE
Is there a way to make dbeaver UI readable on KDE dark mode

### DISCUSSION THREAD

**Discussion by danielwestermann** [dbeaver/dbeaver#13425] [Q&A] [answered]
Hi,

I am using KDE in dark mode as my graphical user interface on Linux. The dbeaver UI looks like this and it is almost not usable:

![image](https://user-images.githubusercontent.com/13865881/67459278-7b5ffb00-f638-11e9-9c47-84c8aa822e92.png)

Is there an option (I did not find one) to make this looking better?

dbeaver version is: 6.2.3

Thanks in advance
Daniel

**Comment by implmnt** [comment]
Same problem

**Comment by nicoschl** [comment]
This is not a native fix but a workaround I use. And it is a slight pain because you have to install the plugin again after each upgrade. You can install the "DevStyle Darkest Theme" plugin from http://www.genuitec.com/updates/devstyle/ci/

**Comment by danielwestermann** [comment]
Hi Nico,

thanks for the work around, but, as you said, that seems not be a solution.

Cheers,
Daniel


________________________________

**Comment by serge-rider** [comment, marked as answer]
You could try to install Darkest Dark theme, on some systems it is much better. See main menu Help->Install new software->Choose "Darkest Dark Theme" in drop down.

**Comment by danielwestermann** [comment]
That is much better, thank you

________________________________

**Comment by hnavia** [comment]
> You could try to install Darkest Dark theme, on some systems it is much better. See main menu Help->Install new software->Choose "Darkest Dark Theme" in drop down.

Unfortunately, Darkest Dark Theme is not working anymore in DBeaver 21.1.0.

Everytime I install the theme, DBeaver enters in a boot loop that never ends.

**Comment by ASHspace** [comment]
> Unfortunately, Darkest Dark Theme is not working anymore in DBeaver 21.1.0.
> 
> Everytime I install the theme, DBeaver enters in a boot loop that never ends.

I am using flatpak **Version 21.1.3.202107181810** and Darkest Dark working fine.

**Comment by hnavia** [comment]
> > Unfortunately, Darkest Dark Theme is not working anymore in DBeaver 21.1.0.
> > Everytime I install the theme, DBeaver enters in a boot loop that never ends.
> 
> I am using flatpak **Version 21.1.3.202107181810** and Darkest Dark working fine.

Yeah, the solution was to install DBeaver for a single user rather than all

**Comment by mcabreb** [comment]
Hi, I hope this is not necroposting but I have the same problem. I had gnome and somehow all my icons were corrupted and was not able to restore them. I switch to kde and it all works well except for some gnome based apps, including dbeaver. The main problem I have is I cannot see any checkboxes in the application. And it makes it quite difficult to use. I tried using the darkest dark theme and it simply changes some colors but the checkboxes are still invisible, I guess they always use the same color for fore and back. Any ideas on how to change this cfg in dbeaver? And any hints on how to completely restore gnome icons would also be great. Thanks!
